CHANGELOG.md

```diff
@@ -0,0 +1,240 @@
+# Changelog
+
+## [Unreleased]
+
+### Added
+* Travis: add 9 (@ralphtheninja)
+
+### Changed
+* Update `encoding-down` to `~4.0.0` (@ralphtheninja)
+* Update `leveldown` to `^3.0.0` (@ralphtheninja)
+
+### Fixed
+* Test: clean up `level-test-*` dbs after tests are done (@ralphtheninja)
+
+## [2.1.0] - 2017-12-13
+
+### Added
+* Add `standard` for linting (@ralphtheninja)
+
+### Changed
+* Attach `.errors` from `levelup` to `Level` constructor (@ralphtheninja)
+
+## [2.0.2] - 2017-11-11
+
+### Changed
+* Update `encoding-down` to `~3.0.0` (@vweevers)
+* README: update node badge (@vweevers)
+
+### Fixed
+* Travis: restore node 4 (@vweevers)
+
+## [2.0.1] - 2017-10-12
+
+### Added
+* Test that encoding options default to utf8 (@ralphtheninja)
+* Test that `.keyEncoding` and `.valueEncoding` are passed to `encoding-down` (@ralphtheninja)
+
+### Fixed
+* Fix encoding options to `encoding-down` (@ralphtheninja)
+
+## [2.0.0] - 2017-10-11
+
+### Added
+* README: add `level` badge (@ralphtheninja)
+
+### Changed
+* Update `levelup` to `^2.0.0` (@ralphtheninja)
+* Update `encoding-down` to `~2.3.0` (@ralphtheninja)
+* Update `leveldown` to `^2.0.0` (@ralphtheninja)
+* README: update npm badges to similar badge style (@ralphtheninja)
+
+### Removed
+* README: Remove Greenkeeper badge (@ralphtheninja)
+
+## [2.0.0-rc3] - 2017-09-16
+
+### Changed
+* Update `levelup` to `2.0.0-rc3` (@ralphtheninja)
+* Update `leveldown` to `^1.8.0` (@ralphtheninja)
+
+## [2.0.0-rc2] - 2017-09-12
+
+### Changed
+* Update `levelup` to `2.0.0-rc2` (@ralphtheninja)
+* Update `encoding-down` to `~2.2.0` (@ralphtheninja)
+
+## [2.0.0-rc1] - 2017-09-04
+
+### Added
+* Travis: add 8 (@ralphtheninja)
+* README: add Greenkeeper badge (@ralphtheninja)
+* README: add node badge (@ralphtheninja)
+
+### Changed
+* README: stear away from `LevelDOWN` to `abstract-leveldown` (@ralphtheninja)
```

Comment:
stear -> steer

Reply:
Damn you are good at finding stuff, even better than me :wink: